src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's Block address in the address book.
+ */
+public class Block {
+	private String value;
```

Comment:
it is a private variable. How can Address.java access the value? Hint: you need a getter method. 

Applicable to all the new classes.